src/Renderer/WebXR.js

```diff
@@ -0,0 +1,72 @@
+/* global XRRigidTransform */
+
+import * as THREE from 'three';
+import { VRButton } from 'ThreeExtended/webxr/VRButton';
+
+async function shutdownXR(session) {
+    if (session) {
+        await session.end();
+    }
+}
+
+const initializeWebXR = (view, options) => {
+    document.body.appendChild(VRButton.createButton(view.mainLoop.gfxEngine.renderer));
+    const xr = view.mainLoop.gfxEngine.renderer.xr;
+    const scale = options.scale || 1.0;
+
+    xr.addEventListener('sessionstart', () => {
+        console.log('Web XR session start'); // eslint-disable-line
+
+        const camera = view.camera.camera3D;
+        const webXRManager = view.mainLoop.gfxEngine.renderer.xr;
```

Comment:
You already have `xr` constant in line 14. Can you use it instead ?

Reply:
Fixed.